spark-thrift-server/1.8/README.md

```diff
@@ -0,0 +1,91 @@
+
+Make sure you have a running Spark Shuffle service in your cluster. Otherwise, simply browse to the Universe app store on your DC/OS UI, select and install the Spark Shuffle package before installing the Spark Thrift Server. Default parameters should be fine.
+
+Browse to the Universe in your DC/OS UI and select the Spark Thrift Server package.
+
+![Install: Find Spark Thrift Server package](img/1-universe.png)
+
+Optionally, you may select "Advanced Installation" to tweak the package parameters, including the memory used for executors and driver, number of cores or number of executors, for example. You can also optionally select to expose the service in Marathon-LB for consumption from outside the cluster by checking the "External Access" checkbox.
+
+![Install: Advanced install parameters](img/2-advanced.png)
+
+Once you've selected the desired configuration parameters, proceed to the installation and wait until the service is in "Running" state
+
+![Install: Running](img/3-running.png)
+
+Assuming the default name (`spark-thrift-server`) and port number (`9000`) are used, the service will be available for other workloads *inside* the cluster in the virtual IP:
+
+```bash
+spark-thift-server.marathon.l4lb.thisdcos.directory:9000
+```
+
+If "external access" is selected (as it is by default), the service will also be available in port 9000 of the public node if running Marathon-LB. If your public node is being firewalled, please make sure to allow traffic to port 9000 (or whichever port you select in Advanced Install) to it.
+
+## Usage
+
+You can use the Spark Thrift Server from workloads running inside the cluster of outside of it (if "external access" was left selected). This example shows to to use the Beeline JDBC client to check the service correct functioning.
+
+If your cluster is running Marathon-LB, you can access the service from a computer with IP connectivity to the public node on port 9000. In order to use the Beeline JDBC client to test the Spark Thrift Server in any node where the Docker runtime is installed, you can use the following command:
+
+```bash
+export PUBLIC_NODE_IP=<enter your public node's IP address here>
+docker run --rm -ti --net=host --entrypoint=./beeline.sh sutoiku/beeline:hive-1.2.0 -u jdbc:hive2://$PUBLIC_NODE_IP:9000/default
+
+```
+
+If external access is not enabled, or if you'd rather test access to the Spark Thrift Server from inside the cluster, you can run from any master or agent node in the cluster (assuming a default service name of "spark-thrift-server":
+
+```bash
+docker run --rm -ti --net=host --entrypoint=./beeline.sh sutoiku/beeline:hive-1.2.0 -u jdbc:hive2://spark-thrift-server.marathon.l4lb.thisdcos.directory:9000/default
+
+```
+
+Either way, once the docker image has been downloaded, you should see an output similar to the following
+
+```bash
+$ docker run --rm -ti --net=host --entrypoint=./beeline.sh sutoiku/beeline:hive-1.2.0 -u jdbc:hive2://$PUBLIC_NODE_IP:9000/default
```

Comment:
This has been tested successfully for 1.8 (customer) and 1.9 (myself) with Marathon-LB. There is no reason why 1.8 should be any different, neither with the MLB connection nor with the package itself. Can you please re-test?